src/tasks/components/TaskCard.tsx

```diff
@@ -264,6 +270,7 @@ export class TaskCard extends PureComponent<
 const mdtp = {
   onAddTaskLabel: addTaskLabel,
   onDeleteTaskLabel: deleteTaskLabel,
+  setTasksPageAsCurrent,
```

Comment:
after removing the thunk, this should simply change to `setCurrentTasksPage`. then you'd call it like

```ts
setCurrentTasksPage('TasksPage')
```